front/app/translations/admin/en.json

```diff
@@ -1176,6 +1177,7 @@
   "app.containers.AdminPage.ProjectEdit.surveyServiceTooltipLinkText": "here",
   "app.containers.AdminPage.ProjectEdit.survey_monkey": "Survey Monkey",
   "app.containers.AdminPage.ProjectEdit.survey_xact": "SurveyXact",
+  "app.containers.AdminPage.ProjectEdit.tagIsLinkedToStaticPage": "This tag cannot be deleted because it is being used to display projects on the following more custom page(s). \nYou will need to unlink the tag from the page, or delete the page before you can delete the tag.",
```

Comment:
I onced used `{br}` to print new line https://github.com/CitizenLabDotCo/citizenlab/blob/08255adb16406073f72da1f68927784f45cd4251/front/app/components/SignUpIn/SignUp/messages.ts#L139 (and then replacing it with `<br>`).

I suppose `\n` (and I guess `white-space: pre-wrap`) is better?

P.S. Do we need `\n` in `areaIsLinkedToStaticPage`?

Reply:
It's actually unnecessary to add a line break, I removed it from both